wagtail/admin/api/views.py

```diff
@@ -19,6 +21,7 @@ class PagesAdminAPIViewSet(PagesAPIViewSet):
 
     actions = {
         'copy': CopyPageAPIAction,
+        'translate': CopyForTranslationAPIAction,
```

Comment:
For consistency with the class/method name, could we keep this ``copy-for-translation``?